cmd/gen-go-sample/inittree.go

```diff
@@ -155,10 +173,10 @@ func (t *initTree) get(k string, info pbinfo.Info) (*initTree, error) {
 							}
 						}
 					}
+					v.typ.repeated = false
```

Comment:
I'm curious if this is necessary. Per protobuf3 [docs](https://developers.google.com/protocol-buffers/docs/proto3#maps), maps   cannot be repeated. I'd have to assume this case is impossible as this tool consumes a descriptor and protoc would fail if it encountered a `repeated map` field. The above checking for the repeated label (and the following assignment) would produce the necessary `false` value.

Does that track or am I missing a scenario?